python/core-workflow#339: Add "Python triage team" section to "triaging.rst" (#506)
python/devguide: triaging.rst

Contributors with the Developer role on the `issue tracker`_ can triage issues
directly without any assistance.

Additionally, this section provides an overview of the bug triager role.

> Bug Triager

Reviewer: ```suggestion
Python triage team
```

> -----------------------------

Reviewer 2: Please shorten underline to match heading.

Author: @willingc What is the length requirement of the underline in comparison to the title length in reST? When the title length was increased, I received build errors due to the previous one being too short.

Author: Based on the other title underlines, it looks the the end of the title must be on the same character position in which the last word of the header starts (at minimum). Let me know whether or not this is acceptable.

Reviewer 2: The underline should be the same as the line above it.

```
Python triage team
-------------------
```

Author: Oh my apologies, for some reason I was basing it on the visual length, not the actual number of characters. Fixed.

> The bug triager is a role dedicated towards improving workflow efficiency and

Reviewer: ```suggestion
The Python triage team is a role dedicated towards improving workflow efficiency and
```

> efficiency and lessening the workload of the core developers. The expectations of

Reviewer 2: ```suggestion
efficiency through thoughtful review and triage of open issues and pull requests. This helps contributors receive timely feedback and enables core developers to focus on reviewed items which reduces their workload. The expectations of
```

> expand upon the "python triager" role on the `issue tracker`_. The responsibilities

Reviewer: In bpo, the triage role is currently called "Developer" role. 🤔 I'm not sure whether we want to refer to it as .. _" `Developer` role in the issue tracker"_ or _" bug triage role in the issue tracker"_

Author: I think calling the triage role on bpo the "Developer" role would probably help to avoid confusion. The two roles have a similar purpose of improving workflow efficiency, but based on the discussions and outline of the role thus far, it seems that the GitHub triager role differs a bit in terms responsibilities. I'll adjust it to "Developer".

listed below are primarily centered around the Python GitHub repositories. This

> extends beyond cpython, to the other repos such as devguide, core-workflow, and

Reviewer: ```suggestion
extends beyond CPython, to the other repos such as devguide, core-workflow, and
```

> extends beyond cpython, to the other repos such as devguide, core-workflow, and

Reviewer: Since current bug triagers are only triaging CPython repo, I'm unsure if we want to grant triage role to the other repos, or just CPython for now. I've asked in https://github.com/python/core-workflow/issues/334#issuecomment-513282372 for opinion.

Author: Personally, I've of the opinion that it should include any `Python core` repository, especially devguide and core-workflow. I expanded upon this a bit on the [issue page](https://github.com/python/core-workflow/issues/334).

> repositories. This extends beyond CPython, to the other repos such as devguide

Reviewer 2: ```suggestion
repositories. This extends beyond CPython, and, as needed, to other repos such as devguide
```

> python.org.

Reviewer: Python.org is not a repo under `Python core` team, let's remove it.

> * PR/issue management

Reviewer 2: ```suggestion
Responsibilities include:

* PR/issue management
```

    - Renaming PRs
    - Reviewing PRs
    - Closing PRs and issues
    - Assisting contributors
    - Notifying appropriate core developers
* Applying appropriate labels to PRs/Issues
    - Skip news
    - Skip issue
    - Good first issue
    - Other categorizations

It is also of paramount importance to treat every contributor to the Python
project kindly and with respect. Regardless of whether they're entirely new
or a veteran core developer, they're actively choosing to voluntarily donate their
time towards the improvement of Python. As is the case with any member of
the Python Software Foundation, always follow the `PSF Code of Conduct`_.

.. note::

>    Existing members of the "Developer" role on the `issue tracker`_ are welcome to

Reviewer 2: ```suggestion
   Existing members with the "Developer" role on the `issue tracker`_ are welcome to
```

>    self-nominate for the bug triager role if they are interested.

Reviewer 2: ```suggestion
   self-nominate for triage team membership if they are interested.
```

Author: Thanks, forgot to update that one.

* (Optional) Leave a brief comment about the proposed next action needed. If
  there is a long message list, a summary can be very helpful.

Bug Triager

> ---------------------------

Reviewer: ```suggestion
------------
```
